optuna/storages/_rdb/storage.py

```diff
@@ -145,6 +145,21 @@ def objective(trial):
         https://docs.sqlalchemy.org/en/13/core/engines.html#sqlalchemy.create_engine.params.
         pool_pre_ping
 
+    .. note::
+        Mainly in a cluster environment, running trials are often killed unexpectedly by the job
```

Comment:
`by the job scheduler` may be unnecessary (based on the discussion https://github.com/optuna/optuna/pull/3229#discussion_r785289561).